mtags-shared/src/main/scala/scala/meta/internal/pc/HoverMarkup.scala

```diff
@@ -2,6 +2,8 @@ package scala.meta.internal.pc
 
 object HoverMarkup {
 
+  private val MaxHoverBodyLength = 50000
```

Comment:
Is that based on some VS Code standard? If so could you add a link to where you found it?

Reply:
I didn't find any information about this in docs. It's based solely on experience, that the limit is around 100000.

Reply:
Could you add a comment about that then? That it's based on the experiment as of day x.y.z